### PR TITLE
chore: update action

### LINKED ISSUE #3: Better release with release that

### LINKED ISSUE #4: Update doc after joining actions-able

### LINKED ISSUE #5: Add debug mode parameter

### PR DESCRIPTION
**Pull request by fhgbaguidi**
closes #5 
closes #4 
closes #3

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -12,7 +12,7 @@ This action substitute the values of environment variables using envsubst. envsu
           fetch-depth: 0
 
       - name: Substitute env
-        uses: fhgbaguidi/envsubst-action@v1.0.0
+        uses: actions-able/envsubst-action@v1.0.1
```
Comment (rlespinasse) [inline README.adoc]: With the `release that` use, you can use `v1` instead

File: entrypoint.sh [hunk 2]
```diff
@@ -3,15 +3,15 @@
...
 OUTPUT_FILE=$3
 INPUT_FILE_PATTERN=$4
 OUTPUT_FILE_SUFFIX=$5
+DEBUG=$5
 
 verbose() {
-  if [ "${DEBUG}" = true ]; then
+  if [ "${DEBUG}" = 'true' ]; then
     echo "${1}"
   fi
```
Comment (rlespinasse) [inline entrypoint.sh]: Consider using the built-in debug message : https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-a-debug-message

```suggestion
  echo "::debug::${1}"
```

**Review by rlespinasse — APPROVED** [review]

**Comment by elijah** [comment]
Bump.

**Comment by rlespinasse** [comment]
With @fhgbaguidi we decided to split the PR and this one is becoming obsolete.

**Pull request closed without merging**